### PR TITLE
Change CPEs detailspage to use GraphQL

### PR DESCRIPTION
**Pull request by y0urself**
**What**:

* Change CPEs detailspage to use GraphQL

**Why**:

**How**:

**Checklist**:

- [x] Tests
- [x] [CHANGELOG](https://github.com/greenbone/gsa/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

File: gsa/src/gmp/parser.js [hunk 1]
```diff
@@ -21,6 +21,9 @@ import {isEmpty} from 'gmp/utils/string';
 
 import date, {duration} from 'gmp/models/date';
 
+export const parseScoreToSeverity = value =>
```
Comment (bjoernricks) [inline gsa/src/gmp/parser.js:24]: The function should get some unitttests

File: gsa/src/gmp/models/cpe.js [hunk 2]
```diff
@@ -22,24 +22,36 @@ import {map} from 'gmp/utils/array';
...
+import {parseScoreToSeverity, parseSeverity, parseDate} from 'gmp/parser';
 
 class Cpe extends Info {
   static entityType = 'cpe';
 
+  static parseObject(object) {
+    const ret = super.parseObject(object);
+
+    ret.severity = parseScoreToSeverity(ret.score);
+    delete ret.score;
```
Comment (bjoernricks) [inline gsa/src/gmp/models/cpe.js:34]: I am really not sure about this at all. The severity will be only the displayed value of the score in future. Maybe we also switch to display the score directly. So deleting the score feels wrong to me but for the moment it seems the best to do.

Comment (y0urself) [reply gsa/src/gmp/models/cpe.js:34]: Yeah I introduced the function that itself cares about dividing the score.
I am with you that we could just show the integer directly ...

File: gsa/src/web/pages/cpes/detailspage.js [hunk 3]
```diff
@@ -237,6 +294,6 @@ CpePage.propTypes = {
 export default withEntityContainer('cpe', {
```
Comment (saberlynx) [inline gsa/src/web/pages/cpes/detailspage.js:294]: Would it still work if you just exported Page? Should work IMO.

File: gsa/src/web/pages/cpes/detailspage.js [hunk 4]
```diff
@@ -159,74 +148,142 @@ Details.propTypes = {
...
+            );
+          }}
+        </EntityPage>
+      )}
+    </EntityComponent>
+  );
+};
 
-CpePage.propTypes = {
+Page.propTypes = {
```
Comment (saberlynx) [inline gsa/src/web/pages/cpes/detailspage.js:286]: Page does not seem to have any props anymore.

**Review by bjoernricks — APPROVED** [review]

**Comment by bjoernricks** [comment]
@y0urself could you just resolve the conflict in the CHANGELOG file? Afterwards the PR can be merged :-)

**Review by bjoernricks — APPROVED** [review]

**Pull request merged**